app/src/ui/blank-slate/create-tutorial-repository-dialog.tsx

```diff
@@ -88,7 +88,7 @@ const InititalReadmeContents =
   `# Welcome to GitHub Desktop!${nl}${nl}` +
   `This is your README. READMEs are where you can communicate ` +
   `what your project is and how to use it.${nl}${nl}` +
-  `Make any change to this file, save it, and then head ` +
+  `Write your name on line 6, save it, and then head ` +
```

Comment:
I love the specificity here! Will the line numbers be consistent across editors? In other words, can we rely on this "line 6" to make sense, or should we use something like, "Write your name on a new line after this one, save it..."?

Reply:
I _think_ we can rely on "line 6" but maybe @outofambit can confirm?

Reply:
"line 6" should be consistent across text editors!